lib/ruby/lib/alces/stack/boot/run.rb

```diff
@@ -37,7 +37,10 @@ def initialize(options={})
           @finder = Alces::Stack::Templater::Finder.new("#{ENV['alces_BASE']}/etc/templates/boot/", options[:template])
           @group = options[:group]
           @dry_run_flag = options[:dry_run_flag]
+          @permanent_boot_flag = options[:permanent_boot_flag]
```

Comment:
`@template_parameters` would be an instance variable contained within the instance of the `Options` class, addressable via `options.template_parameters`.  Is that what you mean?